clusters/nerc-ocp-test/koku-metrics-operator/application.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: koku-metrics-operator
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/nerc-ocp-config.git
+    targetRevision: HEAD
+    path: koku-metrics-operator/overlays/nerc-ocp-test
+  destination:
+    name: nerc-ocp-test
+    namespace: koku-metrics-operator
+  syncPolicy:
```

Comment:
Unless there's a specific reason this needs to be different, **do not** set the `syncPolicy` here in the application; instead, add the `nerc.mghpcc.org/sync-policy: common` label to the resource so the policy is added by the parent `kustomization.yaml`

Reply:
Ah, forgot about that patch, updated